tests/execution/test-call-operator-errors.md

```diff
@@ -0,0 +1,32 @@
+# test-call-operator
+
+###### sdl error
+
+#### server:
+
+```graphql
+schema @server @upstream(baseURL: "http://localhost:3000") {
+  query: Query
+}
+
+type Query {
+  posts: [Post] @http(path: "/posts")
+  userWithoutResolver(id: Int!): User
+  user(id: Int!): User @http(path: "/users/{{args.id}}")
+  userWithGraphQLResolver(id: Int!): User @graphQL(name: "user", args: [{key: "id", value: "{{args.id}}"}])
+  userWithGraphQLHeaders(id: Int!): User @graphQL(name: "user", headers: [{key: "id", value: "{{args.id}}"}])
+}
+
+type User {
+  id: Int!
+}
+
+type Post {
+  userId: Int!
+  withoutResolver: User @call(query: "userWithoutResolver", args: {id: "{{value.userId}}"})
+  withoutOperator: User @call(args: {id: "{{value.userId}}"})
+  urlMismatchHttp: User @call(query: "user", args: {})
+  argumentMismatchGraphQL: User @call(query: "userWithGraphQLResolver", args: {})
+  headersMismatchGraphQL: User @call(query: "userWithGraphQLResolver", args: {})
```

Comment:
```suggestion
  headersMismatchGraphQL: User @call(query: "userWithGraphQLResolver")
```
Args should be optional.

Reply:
> Args should be optional.

Changed. The `args` didn't had a default value, but I implemented it on [b199a05](https://github.com/tailcallhq/tailcall/pull/1064/commits/b199a05a64b910cf96a45a15edf3c4a2f6eab519)

> Error when no args are supplied

This is already being tested. The snapshot for this error is on [tests/snapshots/execution_spec__test-call-operator-errors.md_errors.snap](https://github.com/ologbonowiwi/tailcall/blob/feat/%23846/create-call-operator/tests/snapshots/execution_spec__test-call-operator-errors.md_errors.snap)

Reply:
> Changed. The args didn't had a default value, but I implemented it on [b199a05](https://github.com/tailcallhq/tailcall/pull/1064/commits/b199a05a64b910cf96a45a15edf3c4a2f6eab519)

Re-reading now, I'm not sure if this was the expected solution. Let me know if so. It made sense to make it optional as sometimes no arg is needed, and if it's the case, the user shouldn't be required to declare an "empty" `args` field.